infrastructure/cloudconnect/gcp-vpn.jinja

```diff
@@ -0,0 +1,126 @@
+{#
+Copyright 2017 Google Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+
+{% set id = env["deployment"] %}
+{% set region = properties["region"] %}
+{% set bgp = true if properties["asn"] else false %}
+
+resources:
+  - name: {{ id }}
```

Comment:
can you not indent each element of a list?

That is write: 


resources:
- name: {{id }
  type: ...
- name: blah
...

rather then 

resources:
  - name: ...
    type: ...


the extra whiespace to indent each line elements isn't necessary and can confuse readers. This is also the format followed with our our DM templates at https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/examples/v2

thanks